src/faebryk/core/parameter.py

```diff
@@ -884,6 +877,28 @@ def __init__(
         self.tolerance_guess = tolerance_guess
         self.likely_constrained = likely_constrained
 
+    @property
+    def within(self) -> Ranges | None:
+        return self._within
+
+    @within.setter
```

Comment:
I guess you need this because you first make the Param then find out about its within&sofset.
Got to make sure that we take the mutable within & softset into account everywhere. Maybe limiting it with '@assert_once' might be a good idea for the moment,

Reply:
I like that idea. It won't work for the front-end because of how overrides currently work in `ato`.
Maybe this deserves another issue to improve safety around later? No one in their right mind should do this without being quite sure of what they're doing